src/containers/WaitForSettings/WaitForSettings.js

```diff
@@ -51,9 +65,16 @@ class WaitForSettings extends Component {
       isPluginSettingsLoaded &&
       isAnalyticsLoaded;
 
-    let link = (
-      <a href={CLOUDFLARE_ADD_SITE_PAGE} target="_blank">Cloudflare</a>
+    var link = (
+      <Link href={CLOUDFLARE_ADD_SITE_PAGE} target="_blank">Cloudflare</Link>
     );
+    if (isDNSPageEnabled(config)) {
+      link = (
+        <Link onClick={() => this.handleClick(DOMAINS_OVERVIEW_PAGE)}>
+          <FormattedMessage id="container.dnsManagementPage.title" />
+        </Link>
+      );
+    }
```

Comment:
I tried to do inline if case thing but prettier was complaining and couldn't figure out why. Then did this instead of `ignore-prettier`